java/libs/src/main/java/org/physical_web/collection/PhysicalWebCollection.java

```diff
@@ -35,7 +35,7 @@
   private static final String SITEURL_KEY = "siteurl";
   private Map<String, UrlDevice> mDeviceIdToUrlDeviceMap;
   private Map<Class, UrlDeviceJsonSerializer> mUrlDeviceTypeToUrlDeviceJsonSerializer;
-  private Map<String, PwsResult> mRequestUrlToPwsResultMap;
+  private Map<String, PwsResult> mBroadcastUrlToPwsResultMap;
```

Comment:
My 2 cents: this is a "resolved url" to "url metadata" map, and we have a separate data structure for "broadcast url" to "resolved url".

The reason for this multiple beacons with different "broadcast urls" may resolve to the same PwsResult, and they should share.

And generally we may want to look up PwsResults for a specific url, without having to know which broadcast url was used to get the result..